packages/home/config.toml

```diff
@@ -1,48 +1,80 @@
     "Please do not stack less than 40 STX. Small payouts are hard to manage.",
     "We start locking your STX around 500 blocks before the start of the next cycle. You can join until 200 blocks before the start. The pool will be closed around 50 blocks before the start.",
     "Also, Friedger Pool will re-stack your STXs for the next cycles without any action from you when you chose to delegate for indefinite duration. If you want to leave the pool, just revoke in Stacks Wallet.",
     "You can track Bitcoin earnings here, which gets converted to STX and paid to you within 2 days after the last reward was received:",
-    "Bitcoin address:", "[**33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq**](https://www.blockchain.com/btc/address/33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq)",
-    "The deprecated address [SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7](https://explorer.stacks.co/address/SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7?chain=mainnet) has reached end of use and Stacks on this address won't be re-stacked. Also the old Bitcoin address [**1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF**](https://www.blockchain.com/btc/address/1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF) won't receive any rewards anymore.",
+    "Bitcoin address (new):", "[**33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq**](https://www.blockchain.com/btc/address/33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq)",
+    "The deprecated address [SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7](https://explorer.stacks.co/address/SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7?chain=mainnet) is still maintained, also the old Bitcoin address [**1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF**](https://www.blockchain.com/btc/address/1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF)",
     "More FAQs about delegated stacking at [stacks101.com](https://stacks101.com/stx-stacking/delegation)."]
-    resume = "https://www.hiro.so/wallet"
+
+    # Display Pool addresses here   
+    [[params.pool.row]]
+      period = "Locking for 1 cycle:"
+      address = "[**SP1K1A1PMGW2ZJCNF46NWZWHG8TS1D23EGH1KNK60**](https://explorer.stacks.co/address/SP1K1A1PMGW2ZJCNF46NWZWHG8TS1D23EGH1KNK60?chain=mainnet)"
+    # You can make more rows by copy and pasting the code above
+
+    [[params.pool.row]]
+      period =  "Locking for 3 cycles:"
+      address = "[**SP3K3ZEQVE1E914TFPFMT3A7M53MNMWZCFVQCQB0H**](https://explorer.stacks.co/address/SP3K3ZEQVE1E914TFPFMT3A7M53MNMWZCFVQCQB0H?chain=mainnet)"
+
+    [[params.pool.row]]
+      period =  "Locking for 6 cycles:"
+      address = "[**SP6K6QNYFYF57K814W4SHNGG23XNYYE1B8NV0G1Y**](https://explorer.stacks.co/address/SP6K6QNYFYF57K814W4SHNGG23XNYYE1B8NV0G1Y?chain=mainnet)"
+
+    [[params.pool.row]]
+      period =  "Locking for 8 cycles, until cycle #15 (just before block #700000):"
+      address = "[**SP700C57YJFD5RGHK0GN46478WBAM2KG3A4MN2QJ**](https://explorer.stacks.co/address/SP700C57YJFD5RGHK0GN46478WBAM2KG3A4MN2QJ)"
+    
+
+[params.info_pane]
```

Comment:
This should be higher up on the page.

Reply:
I guess, that instead of scrolling, best if we have a dedicated page for the Pool info. Then we can have all the info right at the top.

Reply:
new Pool page in next PR